Webapp - Beer methods

rayolanderos opened the pull request:
General WebApp layout is set

Beer methods available: 
- Add
- Delete
- Update
- View All
- View Single

Known bug:
- Dealing with the uploading of images has proved tricky. It currently only works if you upload an image on both buckets each time when updating / creating a beer.

rayolanderos committed:
Tap list markup

-Tap list markup
- Mock services
- Beer model
- Uploader styles
- Uploader scripts

rayolanderos committed:
Successful Beer creation

- adds name, price, abv, ibus, description, srm, on tap
- missing: images

rayolanderos committed:
Update Beer

Known bugs: Second upload is slow -- needs better upload detection.

rayolanderos committed:
Delete Beer

deletes beer
added confirmation messages to layout

rayolanderos committed:
Get single beer

Gets single beer info and returns json

gleono commented on webapp/services/get_beer.py:
```diff
@@ -0,0 +1,37 @@
+
+		if beer != None:
+			beer_data = { 
+				"beerId" : beer.key.id(),
```
Would be nice to have the same keys on both methods `get_all_beers` and `get_beer`. I prefer the format used in get_all_beers. Also I think is important to return the beer review and review_count also here since I think they'll be used on the app.

rayolanderos replied:
You are right.. I did a lazy copy paste from the html methods. Should have checked twice.

gleono commented on webapp/services/get_all_beers.py:
```diff
@@ -0,0 +1,43 @@
+		    	'price' : beer.price, 
+		    	'abv' : beer.abv, 
+		    	'ibus' : beer.ibus, 
+		    	'srm' : beer.srm, 
```
What is srm?

gwoodard3 replied:
Looks like beer color

rayolanderos replied:
yeah it's got to do with the tone (light/dark)

gleono reviewed (commented):
Looks great! Only consider addressing the comments before submitting :)

rayolanderos committed:
Updated get single beer method

Get single matches keys in get all beers. Added review, review count, and creation date.
